tests/pipelines/stable_diffusion_xl/test_stable_diffusion_xl.py

```diff
@@ -160,6 +160,7 @@ def test_stable_diffusion_xl_negative_prompt_embeds(self):
         negative_prompt = 3 * ["this is a negative prompt"]
         inputs["negative_prompt"] = negative_prompt
         inputs["prompt"] = 3 * [inputs["prompt"]]
+        inputs["num_images_per_prompt"] = 2
```

Comment:
I think instead of modifying this test case, it'd be better to have a separate test case that checks the validity of your changes in isolation.

Reply:
yeah similar to these tests:

https://github.com/huggingface/diffusers/blob/2e53936c97d167713c9e97414160124861fa4b68/tests/pipelines/stable_diffusion/test_stable_diffusion.py#L221

https://github.com/huggingface/diffusers/blob/2e53936c97d167713c9e97414160124861fa4b68/tests/pipelines/stable_diffusion/test_stable_diffusion.py#L257